docs/FrontEndGuide.txt

```diff
@@ -0,0 +1,4 @@
+In the Angular Project in src\environments\enviroment
+
+the programmer needs to change the baseURL to where its server for the
+API's its located
```

Comment:
Thanks. Documentation is important. I'd only changed that last "its" to "is" 
-> "... to where its server for the API **is** located."